readme.md

```diff
@@ -1,51 +1,153 @@
-# Ewon Connector
+
+# Ignition Ewon Connector
+
+## [Table of Contents](#table-of-contents)
```

Comment:
I think a small link at the bottom of each section that says "Back to top" or something along those lines might work better, since theres really no indication that the section headers link back up to the table of contents.